examples/text-generation/utils.py

```diff
@@ -165,14 +165,19 @@ def get_torch_compiled_model(model):
     return model
 
 
-def setup_quantization(model, args):
+def setup_quantization(model, args=None):
     if os.getenv("USE_INC", ""):
         from neural_compressor.torch import FP8QuantConfig, convert, prepare
-        config = FP8QuantConfig.from_json_file(args.quant_config)
-        if config.calibrate:
-            model = prepare(model, config)
-        elif config.quantize:
+        if args:
+            config = FP8QuantConfig.from_json_file(args.quant_config)
```

Comment:
need to adjust to latest INC API,
see https://github.com/HabanaAI/optimum-habana-fork/commit/2c5193e8d1b1031af140006491dda60e4bb5006b